### ISSUE TITLE
Sharing codes

### ISSUE THREAD

**Issue by ghost** [jiewwwang/Single-cell-retinal-regeneration#2]
Hi, 

This is an very interesting paper. I notice that you used a variety of methods to analyze trajectories. Are you planning to share the resource codes here?

Thanks!

**Comment by adf-ncgr** [comment]
Agreed- I'd very much like to see your analysis scripts posted too! The paper indicates this repository should have them, so maybe it's just an oversight?

**Comment by jiewwwang** [comment]
The codes will be released very soon. We spent time to arrange them into R package. Thanks.

**Comment by adf-ncgr** [comment]
great, thanks! will look forward to seeing them when ready

**Comment by xiaowenchenjax** [comment]
Hi there,
Where is the codes for this paper? are they released ? thanks

**Comment by adf-ncgr** [comment]
per some recent updates to the README it looks like the code has been placed in a different location: https://github.com/jiang-junyao/IReNA 
I hope the authors will kindly correct me if I've misinterpreted!

**Comment by xiaowenchenjax** [comment]
> per some recent updates to the README it looks like the code has been placed in a different location: https://github.com/jiang-junyao/IReNA I hope the authors will kindly correct me if I've misinterpreted!

Thank you for your help.
